src/Bitcoin/Signer.cpp

```diff
@@ -22,12 +23,17 @@ Proto::TransactionPlan Signer::plan(const Proto::SigningInput& input) noexcept {
 
 Proto::SigningOutput Signer::sign(const Proto::SigningInput& input, std::optional<SignaturePubkeyList> optionalExternalSigs) noexcept {
     Proto::SigningOutput output;
+    if (input.is_it_brc_operation()) {
+        auto serializedInput = data(input.SerializeAsString());
+        Rust::CByteArrayWrapper res = Rust::tw_taproot_build_and_sign_transaction(serializedInput.data(), serializedInput.size());
+        output.ParseFromArray(res.data.data(), static_cast<int>(res.data.size()));
+        return output;
+    }
```

Comment:
Can be done later, it's temporary anyway, when we move bitcoin to Rust we will not need this kind of field anymore, we will be able to detect that automatically, here it's use because I need to call Rust